Demo/src/main/java/com/braintreepayments/demo/PreferredPaymentMethodsFragment.java

```diff
@@ -51,68 +44,41 @@ public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup c
         return view;
     }
 
-    private void initializeFeatureClients(InitializeFeatureClientsCallback callback) {
-        getBraintreeClient(braintreeClient -> {
-            payPalClient = new PayPalClient(braintreeClient);
-            venmoClient = new VenmoClient(braintreeClient);
-            preferredPaymentMethodsClient = new PreferredPaymentMethodsClient(braintreeClient);
-            callback.onResult(null);
-        });
+    private void initializeFeatureClients() {
+        BraintreeClient braintreeClient = getBraintreeClient();
+        payPalClient = new PayPalClient(braintreeClient);
+        venmoClient = new VenmoClient(braintreeClient);
+        preferredPaymentMethodsClient = new PreferredPaymentMethodsClient(braintreeClient);
     }
 
     public void launchPreferredPaymentMethods(View v) {
-        initializeFeatureClients(error -> {
-            preferredPaymentMethodsTextView.setText(getString(R.string.preferred_payment_methods_progress));
-            preferredPaymentMethodsClient.fetchPreferredPaymentMethods(getActivity(), result -> {
-                String formatString = "PayPal Preferred: %b\nVenmo Preferred: %b";
-                preferredPaymentMethodsTextView.setText(
-                        String.format(formatString, result.isPayPalPreferred(), result.isVenmoPreferred()));
-
-                billingAgreementButton.setEnabled(result.isPayPalPreferred());
-                singlePaymentButton.setEnabled(result.isPayPalPreferred());
-                venmoButton.setEnabled(result.isVenmoPreferred());
-            });
+        initializeFeatureClients();
+        preferredPaymentMethodsTextView.setText(getString(R.string.preferred_payment_methods_progress));
+        preferredPaymentMethodsClient.fetchPreferredPaymentMethods(getActivity(), result -> {
+            String formatString = "PayPal Preferred: %b\nVenmo Preferred: %b";
+            preferredPaymentMethodsTextView.setText(
+                    String.format(formatString, result.isPayPalPreferred(), result.isVenmoPreferred()));
+
+            billingAgreementButton.setEnabled(result.isPayPalPreferred());
+            singlePaymentButton.setEnabled(result.isPayPalPreferred());
+            venmoButton.setEnabled(result.isVenmoPreferred());
         });
     }
 
     public void launchSinglePayment(View v) {
         getActivity().setProgressBarIndeterminateVisibility(true);
 
-        initializeFeatureClients(initError -> {
-            PayPalCheckoutRequest payPalRequest = createPayPalCheckoutRequest(getActivity(), "1.00");
-            payPalClient.tokenizePayPalAccount(getActivity(), payPalRequest, requestError -> {
-                if (requestError != null) {
-                    handleError(requestError);
-                }
-            });
-        });
```

Comment:
Do we not need to actually launch these anymore?

Reply:
Yeah good call. I added those snippets back in. I used the refactoring tool the IDE clobbered some of the code.